RecoLocalMuon/RPCRecHit/src/RPCRecHitStandardAlgo.cc

```diff
@@ -24,18 +25,32 @@ bool RPCRecHitStandardAlgo::compute(const RPCRoll& roll,
   const float fstrip = (roll.centreOfStrip(cluster.firstStrip())).x();
   const float lstrip = (roll.centreOfStrip(cluster.lastStrip())).x();
   const float centreOfCluster = (fstrip + lstrip)/2;
+  const double y = cluster.hasY() ? cluster.y() : 0;
+  Point = LocalPoint(centreOfCluster, y, 0);
```

Comment:
.. in case someone comes back to this, I drop my earlier question since I have clearly missed a correct answer. The Point is appropriately set to corrected x eventually